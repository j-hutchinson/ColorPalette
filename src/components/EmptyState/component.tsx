import styled from 'styled-components';

const StyledEmptyState = styled.div`
  display: flex;
  justify-content: center;
  font-size: 24px;
  margin: auto: 
`;

const EmptyState = (): JSX.Element => (
  <StyledEmptyState>
    Select a hue now ...
  </StyledEmptyState>
)


export default EmptyState;
