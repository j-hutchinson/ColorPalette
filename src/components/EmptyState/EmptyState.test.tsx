import { shallow } from 'enzyme';
import EmptyState from './component';

describe('EmptyState component', () => {
    test('component matches snapshot', () => {
        expect.assertions(1);

        const wrapper = shallow(<EmptyState />);

        expect(wrapper).toMatchSnapshot();
    });
});