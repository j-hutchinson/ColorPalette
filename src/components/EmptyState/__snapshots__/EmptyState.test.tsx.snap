// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyState component component matches snapshot 1`] = `
<styled.div>
  Select a hue now ...
</styled.div>
`;
